astro/src/content/docs/apis/_webauthn-authenticate-start-request-body.mdx

```diff
@@ -12,11 +14,12 @@ import JSON from 'src/components/JSON.astro';
   <APIField name="credentialId" type="UUID" optional>
     The unique database Id for one of a user's passkeys. When provided, the authentication ceremony will use this passkey. Otherwise, any of a user's passkeys that meet the user verification requirement will be eligible for the authentication ceremony.
   </APIField>
-  <APIField name="loginId" type="String" optional>
-    The email or the username of the authenticating user.
-
-    Required if <InlineField>userId</InlineField> is not present. If both fields are provided in the request body, the <InlineField>userId</InlineField> will be utilized.
-  </APIField>
+  <LoginIdField hide_canonical_details
+                optional
+                user_label="authenticating user"
+                login_id_type_field="loginIdTypes"
+                suffix={<span>Required if <InlineField>userId</InlineField> is not present. If both fields are provided in the request body, the <InlineField>userId</InlineField> will be utilized.</span>} />
```

Comment:
MDX "child components" that are optional seems weird, so passed a 'suffix' property into the login id MDX component.

Reply:
maybe "footer" or "note"? Suffix sounds like something that's going to get appended to the field name.

Reply:
Done